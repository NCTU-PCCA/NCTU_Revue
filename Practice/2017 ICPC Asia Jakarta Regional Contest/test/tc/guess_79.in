9619555145792 44
<<<<<<>>>>><<><<><><><>>><><<<><<<<<<<><>>>>
