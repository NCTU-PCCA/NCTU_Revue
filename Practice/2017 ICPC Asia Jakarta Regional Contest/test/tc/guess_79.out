9619555145792 9619555145791 9619555145790 9619555145789 9619555145788 9619555145787 1 2 3 4 5 9619555145786 9619555145785 6 9619555145784 9619555145783 7 9619555145782 8 9619555145781 9 9619555145780 10 11 12 9619555145779 13 9619555145778 9619555145777 9619555145776 14 9619555145775 9619555145774 9619555145773 9619555145772 9619555145771 9619555145770 9619555145769 15 9619555145768 16 17 18 19
